posthog/session_recordings/session_recording_api.py

```diff
@@ -630,12 +630,11 @@ def snapshots(self, request: request.Request, **kwargs):
         source = request.GET.get("source")
```

Comment:
Doing this plus changing the if/elif on line 654 would be a simpler change imo

```suggestion
        source = request.GET.get("source") or "listing"
```

Reply:
Ah, yeah... it's the difference between what we log and what the value really is 🤔